spec/validators/fee/misc_fee_validator_spec.rb

```diff
@@ -45,6 +45,36 @@
       end
     end
 
+    context 'when misc fee is an evidence provision fee' do
+      let(:fee_type) { build :misc_fee_type, :mievi }
+
+      before { allow(fee).to receive(:fee_type).and_return(fee_type) }
+
+      context 'when the amount is set to' do
```

Comment:
if you put a `subject { fee }` here, you could use `is_expected.to be_valid` below, but either way